integreat_cms/cms/forms/events/event_filter_form.py

```diff
@@ -82,35 +83,21 @@ class EventFilterForm(CustomFilterForm):
 
     query = forms.CharField(required=False)
 
-    def apply(
-        self,
-        events: EventQuerySet,
-        region: Region,
-        language_slug: str,
-    ) -> tuple[EventQuerySet, None, None]:
+    def filter_events_by_time_range(self, events: EventQuerySet) -> EventQuerySet:
         """
-        Filter the events according to the given filter data
+        Filter events by time range
 
-        :param events: The list of events
-        :param region: The current region
-        :param language_slug: The slug of the current language
-        :return: The filtered list of events, the poi used for filtering, and the search query
+        :param events: the unfiltered events
+        :param events: the filtered events
         """
-        if not self.is_enabled:
```

Comment:
I did some digging as to why tests are failing (e.g. test_view_status_code_12) and found that some EventFilterForms that are not "enabled" have no "events_time_range" in self.cleaned_data, thus throwing a KeyError here

Reply:
Interesting! Thank you very much!
Please feel free to commit to this branch if you find a fix :)

But aside from that I have one question. So, I still have a check [if events are enabled](https://github.com/digitalfabrik/integreat-cms/pull/3410/files#diff-10626a88b853c33e85394820b84a767b701d3b51fc0a2e9b61d4f56403a93b2eR233-R234), which is the same except for the return statement. Do you think that's the problem?